src/QMCHamiltonians/ACForce.cpp

```diff
@@ -63,9 +68,12 @@ bool ACForce::put(xmlNodePtr cur)
   attr.add(useSpaceWarp_, "spacewarp", {false}); //"yes" or "no"
   attr.add(swpow, "swpow");                      //Real number"
   attr.add(delta_, "delta");                     //Real number"
+  attr.add(reg_epsilon_,"epsilon");
   attr.add(fastDerivatives_, "fast_derivatives", {false});
   attr.put(cur);
-
+  if( reg_epsilon_ < 0)
+    throw std::runtime_error("ACForce::put(): epsilon<0 not allowed.")
+;
```

Comment:
Semicolon on a separate line?
Please clang-format all the touched files. I saw a few outliers.

Reply:
Oops.  I clang-formatted, but forgot to push up the commit.

Reply:
Done.